docs/specs/cafcass-search-cases.json

```diff
@@ -473,6 +473,188 @@
+            "type": "string",
+            "example": "miamCertificate"
+          },
+          "mpuCertificateByMediator": {
+            "type": "object",
+            "properties": {
+              "document_id": {
+                "type": "string",
+                "format": "uuid",
+                "example": "zzz48b24-c5c2-451c-9fd5-d7a6a87c6355"
+              },
+              "document_filename": {
+                "type": "string",
+                "example": "miamCertificate.pdf"
+              }
+            }
+          },
+          "mpuOtherExemptionReasons": {
+            "type": "string",
+            "example": "miamPolicyUpgradeOtherGrounds_Value_3"
+          },
+          "mpuApplicantUnableToAttendMiamReason1": {
+            "type": "string",
+            "example": "mpuChildProtectionConcern_value_1"
+          },
+          "mpuApplicantUnableToAttendMiamReason2": {
+            "type": "string",
+            "example": "mpuChildProtectionConcern_value_2"
+          },
+          "mpuDocFromDisputeResolutionProvider": {
+            "type": "object",
+            "properties": {
+              "document_id": {
+                "type": "string",
+                "format": "uuid",
+                "example": "zzz48b24-c5c2-451c-9fd5-d7a6a87c6355"
+              },
+              "document_filename": {
+                "type": "string",
+                "example": "disputeResolution.pdf"
+              }
+            }
+          },
+          "miamCertificationDocumentUpload": {
+            "type": "object",
+            "properties": {
+              "document_id": {
+                "type": "string",
+                "format": "uuid",
+                "example": "zzz48b24-c5c2-451c-9fd5-d7a6a87c6355"
+              },
+              "document_filename": {
+                "type": "string",
+                "example": "certificateDocument.pdf"
+              }
+            }
+          },
+          "mpuChildProtectionConcernReason": {
+            "type": "string",
+            "example": "child protection concern"
+          },
+          "mpuMediatorDetails": {
+            "type": "string",
+            "example": "mediator details"
+          },
+          "mpuNoDomesticAbuseEvidenceReason": {
+            "type": "string",
+            "example":"domestic abuse evidence reason"
+          },
+          "miamPolicyUpgradeTable": {
+            "type": "object",
+            "properties": {
+              "mpuApplicantAttendedMiam": {
+                "type": "string",
+                "example": "Yes/No"
+              },
+              "mpuClaimingExemptionMiam": {
+                "type": "string",
+                "example": "Yes/No"
+              },
+              "mpuChildInvolvedInMiam": {
+                "type": "string",
+                "example": "Yes/No"
+              }
+            }
+          },
+          "miamPolicyUpgradeExemptionsTable": {
```

Comment:
mpuIsDomesticAbuseEvidenceProvided

Reply:
@yogendraU  this is string field with Yes/No, it's also in synch with other Yes/No fields, if further clarification needed then please let me know

Reply:
Please remove this field